packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -88,6 +89,20 @@ export class UserService {
     }
   }
 
+  /**
+   * Find Non-Admin Users with given IDs
```

Comment:
The function description is incorrect, the fuction does not do anything explicit to return only non-admin users.